src/encode.cpp

```diff
@@ -277,4 +278,48 @@ operator>>(MessageBuffer& buffer, PublishIntentEnd& msg)
   return buffer;
 }
 
+messages::MessageBuffer&
+operator<<(messages::MessageBuffer& msg, const quicr::Name& val)
+{
+  constexpr uint8_t size = sizeof(quicr::Name::uint_type) * 2;
```

Comment:
how about we add a size() function to Name and that returns the result of above calculation.
```
  constexpr uint8_t size = val.size();
```